### PR TITLE
Add error codes to create variable group

### LINKED ISSUE #1309: Error codes, error chain for variable-group commands
**As a:**
User

**I want:**
error specific codes and error chaining for `spk variable-group` create, command commands

**So that:**
I can understand the root cause.

**Describe the solution you'd like:**
add error code and maintain exception chain

**Acceptance Criteria:**

**Describe alternatives you've considered:**

**Additional context:**

**Does this require updates to documentation?:**
No

part of https://github.com/microsoft/bedrock/issues/1088

### PR DESCRIPTION
**Pull request by edaena**
closes https://github.com/microsoft/bedrock/issues/1309

### CODE REVIEW

File: src/commands/variable-group/create.ts [hunk 1]
```diff
@@ -31,16 +33,20 @@ interface CommandOptions {
 
 export const validateValues = (opts: CommandOptions): void => {
   if (!opts.file) {
-    throw Error("You need to specify a file with variable group manifest");
+    throw buildError(
+      errorStatusCode.VALIDATION_ERR,
+      "variable-group-create-cmd-err-file"
```
Comment (dennisseah) [inline src/commands/variable-group/create.ts]: `variable-group-create-cmd-err-file` -> `variable-group-create-cmd-err-file-missing`

File: src/commands/variable-group/create.ts [hunk 2]
```diff
@@ -31,16 +33,20 @@ interface CommandOptions {
...
+    );
   }
   const config = Config();
   const azure = config.azure_devops;
 
   if (!hasValue(opts.orgName) && !azure?.org) {
-    throw Error(
+    throw buildError(errorStatusCode.VALIDATION_ERR, {
+      errorKey: "variable-group-create-cmd-err-org",
```
Comment (dennisseah) [inline src/commands/variable-group/create.ts]: `variable-group-create-cmd-err-org` -> `variable-group-create-cmd-err-org-missing`

File: src/commands/variable-group/create.ts [hunk 3]
```diff
@@ -49,10 +55,11 @@ export const validateValues = (opts: CommandOptions): void => {
   }
 
   if (!hasValue(opts.devopsProject) && !azure?.project) {
-    throw Error(
+    throw buildError(errorStatusCode.VALIDATION_ERR, {
+      errorKey: "variable-group-create-cmd-err-project",
```
Comment (dennisseah) [inline src/commands/variable-group/create.ts]: `variable-group-create-cmd-err-project` -> `variable-group-create-cmd-err-project-missing`

File: src/commands/variable-group/create.ts [hunk 4]
```diff
@@ -61,12 +68,11 @@ export const validateValues = (opts: CommandOptions): void => {
...
 
   if (!hasValue(opts.personalAccessToken) && !azure?.access_token) {
-    throw Error(
-      `value for ${
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        getCmdOption(decorator, "personal-access-token")!.arg
-      } is missing`
-    );
+    throw buildError(errorStatusCode.VALIDATION_ERR, {
+      errorKey: "variable-group-create-cmd-err-access-token",
```
Comment (dennisseah) [inline src/commands/variable-group/create.ts:72]: `variable-group-create-cmd-err-access-token` -> `variable-group-create-cmd-err-access-token-missing`

File: src/lib/i18n.json [hunk 5]
```diff
@@ -109,6 +109,13 @@
     "ring-delete-cmd-failed": "Error occurred while deleting ring: {0}.",
     "ring-set-default-cmd-failed": "Error occurred while setting default ring: {0}",
 
+    "variable-group-create-cmd-failed": "Error occurred while creating variable group.",
+    "variable-group-create-cmd-err-create": "Could not load variable group. The variable group type '{0}' is not supported. Only 'Vsts' and 'AzureKeyVault' are valid types and case sensitive.",
+    "variable-group-create-cmd-err-access-token": "Please provide a value for {0}.",
+    "variable-group-create-cmd-err-project": "Please provide a value for {0}.",
+    "variable-group-create-cmd-err-org": "Please provide a value for {0}.",
+    "variable-group-create-cmd-err-file": "Please provide a file with the variable group manifest.",
```
Comment (dennisseah) [inline src/lib/i18n.json]: Remove all the `Please`. thanks

**Review by dennisseah — APPROVED** [review]

**Review by samiyaakhtar — APPROVED** [review]

**Pull request merged**